specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2019-06-01-preview/examples/TasksCreate.json

```diff
@@ -0,0 +1,232 @@
+{
+  "parameters": {
+    "subscriptionId": "4385cf00-2d3a-425a-832f-f4285b1c9dce",
+    "resourceGroupName": "myResourceGroup",
+    "registryName": "myRegistry",
+    "api-version": "2019-06-01-preview",
+    "taskName": "mytTask",
+    "taskCreateParameters": {
+      "properties": {
+        "status": "Enabled",
+        "platform": {
+          "os": "Linux",
+          "architecture": "amd64"
+        },
+        "agentConfiguration": {
+          "cpu": 2
+        },
+        "step": {
+          "imageNames": [
+            "azurerest:testtag"
+          ],
+          "dockerFilePath": "src/DockerFile",
+          "contextPath": "src",
+          "isPushEnabled": true,
+          "noCache": false,
+          "arguments": [
```

Comment:
missing `"type": "Docker"` for the validator to validate it as `"DockerBuildStep"` instead of `"TaskStepProperties"`. Hence `"arguments"` is an additional property in the current example.